gnu-apl-osx-workaround.el

```diff
@@ -3,6 +3,8 @@
 (require 'cl-lib)
 (require 'gnu-apl-util)
 
+(defvar gnu-apl--symbols)                   ;gnu-apl-symbols.el
```

Comment:
Why not just `require` here?

Reply:
To be honest, I didn't want to worry about circular dependencies, and because most of the code was required by `gnu-apl-mode.el`, I just declared the variables this way.

Reply:
Unless I'm mistaken, there should be no circular dependency in this case.

Reply:
You are right, that was overcaution on my side.